optaweb-vehicle-routing-frontend/pom.xml

```diff
@@ -97,6 +97,16 @@
               <arguments>run coverage</arguments>
             </configuration>
           </execution>
+          <execution>
+            <id>npm test cypress</id>
```

Comment:
Nope. This fails the CI build.

Reply:
Looks like an issue with missing Xvfb on both Travis and Jenkins; either we use a machine with this library available or the suggested cypress docker image.

Now I realized that even with one of these in place, the test would fail due to backend not being started. I will remove this execution of the frontend plugin for now and let's discuss how to get the backend running together with .osm file in CI.